tests/actions/draftActions.test.js

```diff
@@ -0,0 +1,57 @@
+import { TOGGLE_DRAFT_LAB_ORDER_URGENCY } from '../../app/js/actions/actionTypes';
+
+import toggleDraftLabOrderUrgency from '../../app/js/actions/draftActions';
+
+describe('Draft Actions', () => {
+  beforeEach(() => moxios.install());
+  afterEach(() => moxios.uninstall());
+
+  it(`should dispatch TOGGLE_DRAFT_LAB_ORDER_URGENCY 
+      and set order urgency to ROUTINE if order has no urgency`, async (done) => {
+      const labOrder = { uuid: 'abxzy-123'};
+      const expectedAction = {
+        TOGGLE_DRAFT_LAB_ORDER_URGENCY,
+        order: {
+          ...labOrder, 
+          urgency: 'ROUTINE',
+        },
+      };
+      const store = mockStore({});
+      await store.dispatch(toggleDraftLabOrderUrgency(labOrder), () => {
+        expect(store.getActions()).toEqual(expectedAction);
+      });
```

Comment:
`store.dispatch` is a `Promise` based function and to the best of my knowledge, doesn't take in a second parameter. since you are using the `await` keyword, refactor this to

```
await store.dispatch(toggleDraftLabOrderUrgency(labOrder));
expect(store.getActions()).toEqual(expectedAction);
 ```